data.sql

```diff
@@ -97,3 +97,21 @@ INSERT INTO visits (animal_id, vet_id, visit_date) VALUES
     ((SELECT id FROM animals WHERE name = 'Boarmon'), (SELECT id FROM vets WHERE name = 'Maisy Smith'), '2020-08-03'),
     ((SELECT id FROM animals WHERE name = 'Blossom'), (SELECT id FROM vets WHERE name = 'Stephanie Mendez'), '2020-05-24'),
     ((SELECT id FROM animals WHERE name = 'Blossom'), (SELECT id FROM vets WHERE name = 'William Tatcher'), '2021-01-11');
+
+<<<<<<< HEAD
+
+-- This will add 3.594.280 visits considering you have 10 animals, 4 vets, and it will use around ~87.000 timestamps (~4min approx.)
+INSERT INTO visits (animal_id, vet_id, visit_date) SELECT * FROM (SELECT id FROM animals) animal_ids, (SELECT id FROM vets) vets_ids, generate_series('1980-01-01'::timestamp, '2021-01-01', '4 hours') visit_timestamp;
+
+-- This will add 2.500.000 owners with full_name = 'Owner <X>' and email = 'owner_<X>@email.com' (~2min approx.)
+insert into owners (full_name, email) select 'Owner ' || generate_series(1,2500000), 'owner_' || generate_series(1,2500000) || '@mail.com';
+=======
+-- Add indexes to your visits table animal_id
+CREATE INDEX idx_animal_id ON visits (animal_id);
+
+-- Add indexes to your visits table vet_id
+CREATE INDEX vet_id_asc ON visits(vet_id ASC);
+
+-- Add indexes to your owners table
+CREATE INDEX idx_email ON owners (email);
```

Comment:
[OPTIONAL]
- [ ] This is a nice-to-have suggestion as it won't improve the query that was reported as problematic in the requirements for this project.

  As email addresses are not case-sensitive, we might need to look for the emails in a case-insensitive way. There is an option to reflect that in the index by using:

```sql
CREATE INDEX ON owners ((lower(email)));
```